slasher/db/types/types.go

```diff
@@ -0,0 +1,54 @@
+package types
```

Comment:
Why is types under `/slasher/db`, do you think it should be under `/slasher`?

Reply:
Should I move all types in there then? Types like Service, Config, Server?

Reply:
These types are highly specific only to the db, I think we should just leave them as a subpackage under `slasher/db/types` @terencechain